docs/serving/configuration/feature-flags.md

```diff
@@ -527,3 +528,83 @@ spec:
       schedulerName: custom-scheduler-example
 ...
 ```
+
+### Kubernetes Share Process Namespace
+
+* **Type**: Extension
+* **ConfigMap key:** `kubernetes.podspec-shareprocessnamespace`
+
+This flag controls whether the [share process namespace](https://kubernetes.io/docs/tasks/configure-pod-container/share-process-namespace/) can be used.
+
+```yaml
+apiVersion: serving.knative.dev/v1
+kind: Service
+...
+spec:
+  template:
+    spec:
+      ...
+      shareProcessNamespace: true
+...
+```
+
+!!! warning
+
+    `shareProcessNamespace` and `hostPID` cannot both be set.
```

Comment:
Check https://github.com/kubernetes-client/go/blob/master/kubernetes/docs/V1PodSpec.md for this, probably we should catch this a bit earlier.